# Zoom doesn't get the PocketCHIP screen size right via its query.
# Applying this patch forces it to be correct.
*** src/menu.c	2011-11-05 07:40:53.000000000 -0400
--- src/menu.c.new	2016-12-24 20:38:08.300031582 -0500
***************
*** 151,156 ****
--- 151,159 ----
    qsort(game, n_games, sizeof(struct game_struct), game_compare);
  
    selection = 0;
+   di = display_get_info();
+   di->lines = 18;
+   di->columns = 50;
    height = (di->lines-6)&~1;
    sprintf(format, " %%.%is ", di->columns-6);
  
